x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details_form.tsx

```diff
@@ -9,13 +9,28 @@ import { EuiButtonEmpty, EuiSpacer, EuiText } from '@elastic/eui';
 import React, { memo, useCallback, useState } from 'react';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { MalwareProtections } from './policy_forms/protections/malware';
+import { MemoryProtection } from './policy_forms/protections/memory';
 import { LinuxEvents, MacEvents, WindowsEvents } from './policy_forms/events';
 import { AdvancedPolicyForms } from './policy_advanced';
 import { AntivirusRegistrationForm } from './components/antivirus_registration_form';
 import { Ransomware } from './policy_forms/protections/ransomware';
 import { LockedPolicyCard } from './policy_forms/locked_card';
 import { useLicense } from '../../../../common/hooks/use_license';
 
+const LOCKED_CARD_RAMSOMWARE_TITLE = (
```

Comment:
now sure that we should be using JSX singletons across instance of this component. 
Can you instead convert them to string by using `i18.translate()` instead?

Reply:
@paul-tavares can you elaborate further what do you mean by this?

I am using  `<FormattedMessage` because that's what it is being used everywhere else in this component and children.